jvector-base/src/main/java/io/github/jbellis/jvector/vector/DefaultVectorizationProvider.java

```diff
@@ -43,8 +44,13 @@ public VectorUtilSupport getVectorUtilSupport() {
     return vectorUtilSupport;
   }
 
+  /**
+   * Returns the default vector type support. Hint is not respected.
```

Comment:
"Hint is not respected, because the Default provider only supports on-heap allocations."